packages/server-wallet/src/protocols/process-ledger-queue.ts

```diff
@@ -102,114 +113,135 @@ const xorOutcome = (a: SimpleAllocation, b: SimpleAllocation): SimpleAllocation
   allocationItems: _.xorWith(a.allocationItems, b.allocationItems, areAllocationItemsEqual),
 });
 
-const computeNewOutcome = ({
+const mergeProposedLedgerUpdates = (
+  mine: SimpleAllocation,
+  theirs: SimpleAllocation,
```

Comment:
This is probably totally fine as is, but this could be easily extended to an `n`-party null channel.

I'm not sure if it's worth the abstraction, since the only `n`-party null channel we've come across is the joint channel when virtual funding, and for that, there is no chance of disagreement in the next state update.

It would be worth it if eg. we were to implement some token exchange in a directly funded, 3-party ledger channel.